package.json

```diff
@@ -148,6 +148,6 @@
     "typedoc-plugin-monorepo": "^0.1.0",
     "typescript": "^2.9.2",
     "webpack": "^4.19.1",
-    "webpack-dev-server": "^3.1.8"
+    "webpack-dev-server": "^3.1.9"
```

Comment:
Please don't mix up separate concerns in a single PR. I'll do with the typescript upgrade.

Reply:
Later version of webpack dev server was needed for ui so upgraded it 👀

Reply:
Nothing related to typescript here

Reply:
I meant about version upgrading. Oh, if it's needed then alright. 👍